metaseq/launcher/opt_baselines.py

```diff
@@ -97,6 +96,8 @@ def get_grid(args):
 
     total_gpus = (args.num_gpus * args.num_nodes) // size.model_parallel
     ddp_bsz = (size.batch_size // total_gpus) // SEQ_LEN
+    if args.tp_enabled:
+        ddp_bsz //= 32
```

Comment:
Why do we need to do this? We shouldn't add workarounds like this to fix the OOM but actually figure out the root cause of the OOM and fix it.